ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -56,16 +57,33 @@ Item {
         }
     }
 
-    function clickMessage(isProfileClick, isSticker = false) {
-        if (!isProfileClick) {
-            SelectedMessage.set(messageId, fromAuthor);
+    function clickMessage(isProfileClick, isSticker = false, isImage = false, image = null) {
+        if (!isImage) {
```

Comment:
Could you please do the opposite condition here first. i.e. do `if (isImage)` here and the code of the image popup, then a `return`. That way, we don't need an else and also it makes the condition clearer that this context is **only** for images instead of the context being for "what is not an image", which is confusing

Reply:
I've changed this section to reflect your explanation in the new commit.